[2023-10-25T11:53:50.180+0530] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first_dag.inaccurate scheduled__2023-07-14T00:00:00+00:00 [queued]>
[2023-10-25T11:53:50.186+0530] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first_dag.inaccurate scheduled__2023-07-14T00:00:00+00:00 [queued]>
[2023-10-25T11:53:50.186+0530] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-10-25T11:53:50.195+0530] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): inaccurate> on 2023-07-14 00:00:00+00:00
[2023-10-25T11:53:50.199+0530] {standard_task_runner.py:57} INFO - Started process 50227 to run task
[2023-10-25T11:53:50.201+0530] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'first_dag', 'inaccurate', 'scheduled__2023-07-14T00:00:00+00:00', '--job-id', '1146', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmptimlggiz']
[2023-10-25T11:53:50.202+0530] {standard_task_runner.py:85} INFO - Job 1146: Subtask inaccurate
[2023-10-25T11:53:50.224+0530] {task_command.py:415} INFO - Running <TaskInstance: first_dag.inaccurate scheduled__2023-07-14T00:00:00+00:00 [running]> on host harika-Latitude-5511
[2023-10-25T11:53:50.269+0530] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='first_dag' AIRFLOW_CTX_TASK_ID='inaccurate' AIRFLOW_CTX_EXECUTION_DATE='2023-07-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-07-14T00:00:00+00:00'
[2023-10-25T11:53:50.271+0530] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-10-25T11:53:50.271+0530] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', "echo 'inaccurate'"]
[2023-10-25T11:53:50.279+0530] {subprocess.py:86} INFO - Output:
[2023-10-25T11:53:50.280+0530] {subprocess.py:93} INFO - inaccurate
[2023-10-25T11:53:50.280+0530] {subprocess.py:97} INFO - Command exited with return code 0
[2023-10-25T11:53:50.291+0530] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=inaccurate, execution_date=20230714T000000, start_date=20231025T062350, end_date=20231025T062350
[2023-10-25T11:53:50.332+0530] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-25T11:53:50.343+0530] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
